controls/lambda.sp

```diff
@@ -0,0 +1,36 @@
+locals {
+  lambda_common_tags = merge(local.thrifty_common_tags, {
+    service = "lambda"
+  })
+}
+
+benchmark "lambda" {
+  title         = "Lambda Checks"
+  description   = "Thrifty developers ensure their Lambda functions are optimized."
+  documentation = file("./controls/docs/lambda.md")
+  tags          = local.lambda_common_tags
+  children = [
+    control.high_error_rate,
+    control.excessive_timout,
+  ]
```

Comment:
can you pls sort this and remove 1 comma

```
control.excessive_timout,
control.high_error_rate
```

Reply:
Done